### DISCUSSION TITLE
How does one provide a client side certificate to a request?

### DISCUSSION THREAD

**Discussion by greyarch** [stepci/stepci#56] [Q&A] [answered]
The documentation does not mention how would one make a request with a client side certificate.

**Comment by mishushakov** [comment]
Hey,
Can you give an example how you do this currently?

**Reply by greyarch** [reply]
I am afraid I do not understand the question. Currently, I am not using stepci to do that :)
What I am looking for is the equivalent of the `--cert` and `--key` curl options.

**Reply by mishushakov** [reply]
Ok, thanks for clarification, I get it now

There's certainly a possibility to use client certificates in the http client we're using
https://github.com/sindresorhus/got/blob/main/documentation/5-https.md


I'll keep you updated. Thanks

**Comment by mishushakov** [comment, marked as answer]
Client Certificates are now supported

https://docs.stepci.com/reference/workflow-syntax.html#tests-test-steps-step-http-auth-certificate

**Comment by sebastianwis** [comment]
Hey Greyarch @greyarch 
Thank you for your contribution here! 
It would be amazing to welcome you to our Discord if you haven't joined it yet: 
https://discord.com/invite/KqJJzJ3BTu
This is where we keep users up to date with updates, features and bug resolutions.
